sage-ios/SAGE/SAGE/Semester.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  Semester.swift
+//  SAGE
+//
+//  Created by Andrew Millman on 1/18/16.
+//  Copyright © 2016 Cal Blueprint. All rights reserved.
+//
+
+import Foundation
+
+enum Term: Int {
+    case Spring = 0
+    case Fall = 1
+}
+
+class Semester: NSObject {
+    
+    var id: Int = -1
+    var startDate: NSDate?
+    var term: Term = .Spring
+    
+    //
+    // MARK: - Initialization
+    //
+    init(id: Int = -1, startDate: NSDate? = nil, term: Term = .Spring) {
+        self.id = id
+        self.startDate = startDate
+        self.term = term
+        super.init()
+    }
+    
+    init(propertyDictionary: [String: AnyObject]) {
+        for (propertyName, value) in propertyDictionary {
+            switch propertyName {
+            case SemesterConstants.kId:
+                self.id = value as! Int
+            case SemesterConstants.kStartDate:
+                let formatter = NSDateFormatter()
+                formatter.dateFormat = StringConstants.JSONdateFormat
+                self.startDate = formatter.dateFromString(value as! String)
+            case SemesterConstants.kTerm:
+                self.term = Semester.termFromInt(value as! Int)
+            default: break
+            }
+        }
+        super.init()
+    }
+    
+    //
+    // MARK: - Public Methods
+    //
+    func stringFromStartDate() -> NSString {
```

Comment:
specify that this is display string - rename as displayStringFromStartDate or something